### PR TITLE
Add some tests to new functionality

### PR DESCRIPTION
**Pull request by kirillseva**

### CODE REVIEW

File: tests/testthat/test-routes.r [hunk 1]
```diff
@@ -1,22 +1,33 @@
-context('determine_route')
+context("determine_route")
```
Comment (robertzk) [inline tests/testthat/test-routes.r]: You must use `library(testthatsomemore)` here.

**Comment by robertzk** [comment]
Sweet! Can you add testthatsomemore to .travis.yml and rebase master?

**Comment by robertzk** [comment]
Can you add testthatsomemore to travis.yml?

**Comment by kirillseva** [comment]
it's there already... added it to a bunch of places

**Comment by robertzk** [comment]
I can't fix your test for you because I don't have membership of your forked repo, but I added you to this repo as a contributor so you should be able to move your branch over.

**Comment by robertzk** [comment]
But yeah this is awesome!

**Pull request merged**